clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -184,9 +184,8 @@
                                         Importance.HIGH,
                                         HEARTBEAT_INTERVAL_MS_DOC)
                                 .define(PARTITION_ASSIGNMENT_STRATEGY_CONFIG,
-                                        Type.STRING,
-                                        "range",
-                                        in("range", "roundrobin"),
+                                        Type.CLASS,
+                                        RoundRobinAssignor.class,
```

Comment:
As long as we document the difference, it's probably fine. I can't imagine any cases where users would have a hard dependence on the range assignor. And since a lot of users probably won't override the default, I think we should have it set to the better strategy.